python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -67,6 +74,7 @@ def dbt_assets(
     backfill_policy: Optional[BackfillPolicy] = None,
     op_tags: Optional[Mapping[str, Any]] = None,
     required_resource_keys: Optional[Set[str]] = None,
+    project_dir: Optional[Union[str, Path]] = None,
```

Comment:
We should consider parameterizing this by `project: DbtProject`.

This way, we can envision a future where we deprecate the `manifest` parameter in favor of our new dbt project management scheme.

```python
@dbt_assets(project=DbtProject(project_dir=...))
def my_dbt_assets(...): ...
```